### PR TITLE
Added different ways the potato can be not delicious!

### PR DESCRIPTION
**Pull request by Desetude**

### CODE REVIEW

File: src/main/java/org/drtshock/NonDeliciousReason.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.drtshock;
+
+public enum NonDeliciousReason {
+
+    /**
+     * Different ways to potato can be non-delicious
```
Comment (blha303) [inline src/main/java/org/drtshock/NonDeliciousReason.java]: Different ways **the** potato can be non-delicious

Comment (Desetude) [inline src/main/java/org/drtshock/NonDeliciousReason.java]: shh

**Comment by blha303** [comment]
:+1:

**Comment by jkcclemens** [comment]
Yeah, if this could be squashed and changed to `NotDelicious`, that'd be great.

File: src/main/java/org/drtshock/NonDeliciousReason.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.drtshock;
+
+public enum NonDeliciousReason {
```
Comment (blha303) [inline src/main/java/org/drtshock/NonDeliciousReason.java]: *NotDeliciousReason please

Comment (Desetude) [inline src/main/java/org/drtshock/NonDeliciousReason.java]: Fixed.

**Comment by Ichbinjoe** [comment]
Squash your commits plz <3

**Comment by drtshock** [comment]
I'm not baked right now. Does that mean I'm not delicious? :(

**Comment by blha303** [comment]
Quick tutorial for squashing commits? for reference

**Comment by Ichbinjoe** [comment]
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by Desetude** [comment]
Doing it now

**Comment by blha303** [comment]
`git rebase -i HEAD~5`, change `pick` to `squash` on the second,third,fourth,fifth lines, leaving the first one as `pick`. Then close the editor, it'll prompt you to edit the commit messages. Tbh I just learned how to squash commits just then, thaaaaanks drtshock xoxo. Here's the diff I produced http://s.b303.me/Ppfb.diff

**Comment by Ichbinjoe** [comment]
Your commits will now be much more delicious. And clean. Which everyone likes.

**Comment by jkcclemens** [comment]
Please put different enum values on different lines, as well.

**Comment by Desetude** [comment]
After trying to squash them and trying to push, I get https://gyazo.com/ce2d63555013ea1c4efe329fe9dacab8

**Comment by Desetude** [comment]
Squashed them.

**Comment by blha303** [comment]
:+1:

**Comment by jkcclemens** [comment]
The JavaDocs in `NotDeliciousReason` are wrong. You've documented one of the enum values instead of the class. Ideally, all values and the class should be documented.

**Comment by blha303** [comment]
:-1: :P

**Comment by Desetude** [comment]
Uhh xDdddddddddddd

**Comment by Ichbinjoe** [comment]
rip?

**Comment by Desetude** [comment]
Erm

**Comment by Desetude** [comment]
I didn't close it? xD

**Comment by Desetude** [comment]
I guess I'll recreate this

**Comment by blha303** [comment]
Just reopen it

**Comment by jkcclemens** [comment]
>Ideally, **all values** and the class should be documented.

**Comment by Desetude** [comment]
Keeps on closing for some reason, will reopen again.

**Comment by jkcclemens** [comment]
You're most likely clicking close and comment instead of comment.

**Comment by Desetude** [comment]
I think it's something to do with me syncing, it looks like it's resetting my fork so I need to commit again.

**Comment by Desetude** [comment]
Fingers crossed that there is nothing else to change.

**Pull request merged**